flink-formats/flink-protobuf/src/main/java/org/apache/flink/formats/protobuf/PbConstant.java

```diff
@@ -27,4 +27,10 @@ public class PbConstant {
     public static final String PB_MAP_KEY_NAME = "key";
     public static final String PB_MAP_VALUE_NAME = "value";
     public static final String PB_OUTER_CLASS_SUFFIX = "OuterClass";
+    /**
+     * JIT optimizer threshold is 8K, unicode encode one char use 2byte, so use 3K as
```

Comment:
This is not correct, for ascii chars, there is only 1 byte in unicode encoding.